code/game/gamemodes/gang/dominator.dm

```diff
@@ -20,6 +20,9 @@
 	var/datum/effect_system/spark_spread/spark_system
 	var/obj/effect/countdown/dominator/countdown
 
+/obj/machinery/dominator/hulk_damage()
+	return 20
```

Comment:
do `(max_integrity - integrity_failure) / DOM_HULK_HITS_REQUIRED` and make that a define at the top of the file